### PR TITLE
Booklist

### PR DESCRIPTION
**Pull request by eileenfuerstenau**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import data from '../material/bookdata.json'
```
Comment (RBN2208) [inline src/components/App.js:1]: You could write
` import { books } from '../material/bookdata.json'`
And just write 
`books.map(...`

File: src/components/BookCard.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      </CoverWrapper>
+      <section>
+        <h2>{title}</h2>
+        <Author>{author}</Author>
+      </section>
+    </Card>
+  )
+}
+
+BookCard.propTypes = {
```
Comment (RBN2208) [inline src/components/BookCard.js:19]: Cool!

**Review by RBN2208 — COMMENTED** [review]
Hey! I reviewed your code. Nice work!

File: src/components/BookCard.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  display: grid;
+  grid-template-columns: 2fr 5fr;
+`
+const CoverWrapper = styled.span`
+  display: grid;
+  align-content: center;
+  justify-content: center;
+  padding: 10px;
+`
+const Author = styled.h3`
```
Comment (macfrei) [inline src/components/BookCard.js:37]: Why is it an `h3` element instead of `h1` or `h2`?

File: src/components/BookCard.spec.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import BookCard from './BookCard'
+import { render, screen } from '@testing-library/react'
+
+describe('BookCard', () => {
+  it('renders a booktitle and an author', () => {
```
Comment (macfrei) [inline src/components/BookCard.spec.js:5]: I would test the render of the image as well.

**Review by macfrei — COMMENTED** [review]
Cool! I added a few comments you might want to consider! Keep it up 🚀

File: src/components/App.js (hunk 1 above)
Comment (eileenfuerstenau) [reply src/components/App.js:1]: Changed

File: src/components/BookCard.js (hunk 3 above)
Comment (eileenfuerstenau) [reply src/components/BookCard.js:37]: Thanks, just changed it to an h2 as it is a heading for one element, but not for the entire page

File: src/components/BookCard.spec.js (hunk 4 above)
Comment (eileenfuerstenau) [reply src/components/BookCard.spec.js]: Thank you, i just added the test to check whether the image is rendered

**Pull request merged**